docs/_docs/plugins.md

```diff
@@ -59,14 +60,16 @@ values of the gem names of the plugins you'd like to use. An example:
   </p>
 </div>
 
-In general, plugins you make will fall into one of five categories:
+In general, plugins you make will fall broadly into one of five categories:
 
 1. [Generators](#generators)
 2. [Converters](#converters)
 3. [Commands](#commands)
 4. [Tags](#tags)
 5. [Hooks](#hooks)
 
+See the bottom of the page for available examples of each.
```

Comment:
So let's add the missing navigation item then:

```diff 
++ 6. List of available plugins(#available-plugins)
```

FYI This section is due to move to a specific website (see https://dirtyf.github.io/directory/ for the current WIP)